doc/make.jl

```diff
@@ -113,7 +113,7 @@ const PAGES = [
 ]
 
 makedocs(
-    build     = "_build/html/en",
+    build     = joinpath(pwd(), "_build/html/en"),
```

Comment:
@mortenpi this seems like it should be the implicit default behavior, what's up with this being needed?